packages/plugin-ext/src/main/browser/workspace-main.ts

```diff
@@ -14,23 +14,23 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import * as theia from '@theia/plugin';
-import { interfaces, injectable } from '@theia/core/shared/inversify';
-import { WorkspaceExt, StorageExt, MAIN_RPC_CONTEXT, WorkspaceMain, WorkspaceFolderPickOptionsMain } from '../../common/plugin-api-rpc';
-import { RPCProtocol } from '../../common/rpc-protocol';
+import { CancellationToken, Emitter, Event, ResourceResolver } from '@theia/core';
+import { Disposable, DisposableCollection } from '@theia/core/lib/common/disposable';
+import { Resource } from '@theia/core/lib/common/resource';
+import URI from '@theia/core/lib/common/uri';
+import { injectable, interfaces } from '@theia/core/shared/inversify';
 import { URI as Uri } from '@theia/core/shared/vscode-uri';
-import { UriComponents } from '../../common/uri-components';
 import { FileSearchService } from '@theia/file-search/lib/common/file-search-service';
-import URI from '@theia/core/lib/common/uri';
-import { WorkspaceService } from '@theia/workspace/lib/browser';
-import { Resource } from '@theia/core/lib/common/resource';
-import { Disposable, DisposableCollection } from '@theia/core/lib/common/disposable';
-import { Emitter, Event, ResourceResolver, CancellationToken } from '@theia/core';
-import { PluginServer } from '../../common/plugin-protocol';
 import { FileSystemPreferences } from '@theia/filesystem/lib/browser';
-import { SearchInWorkspaceService } from '@theia/search-in-workspace/lib/browser/search-in-workspace-service';
 import { FileStat } from '@theia/filesystem/lib/common/files';
 import { MonacoQuickInputService } from '@theia/monaco/lib/browser/monaco-quick-input-service';
+import * as theia from '@theia/plugin';
+import { SearchInWorkspaceService } from '@theia/search-in-workspace/lib/browser/search-in-workspace-service';
+import { WorkspaceService } from '@theia/workspace/lib/browser';
+import { MAIN_RPC_CONTEXT, StorageExt, WorkspaceExt, WorkspaceFolderPickOptionsMain, WorkspaceMain } from '../../common/plugin-api-rpc';
+import { PluginServer } from '../../common/plugin-protocol';
+import { RPCProtocol } from '../../common/rpc-protocol';
+import { UriComponents } from '../../common/uri-components';
```

Comment:
@martin-fleck-at do you mind reverting these unnecessary changes, it makes the pull-request harder to review.

Reply:
Ah yes, sorry, I will push an update for that soon.